engine/src/main/java/org/hibernate/validator/BaseHibernateValidatorConfiguration.java

```diff
@@ -427,4 +428,15 @@ default S locales(Locale... locales) {
 
 	@Incubating
 	S beanMetaDataClassNormalizer(BeanMetaDataClassNormalizer beanMetaDataClassNormalizer);
+
+	/**
+	 * Allows setting the Expression Language feature level.
```

Comment:
So... is this the default for all constraints? Or only for default (built-in) constraints? Is it used as the default for custom constraint violations?

I find this a bit confusing...

Reply:
I clarified it in the Javadoc.

I know it's a bit confusing but the defaults are different and I don't want the global default to be the one used by custom violations as they are more sensitive to security issues.